source/eitri/docker_wrapper.py

```diff
@@ -33,28 +33,53 @@
 from retrying import retry
 import psycopg2
 
-# postgres/postgis image
+"""
+This module contains classes about Docker management.
+"""
+
+# Postgres/PostGIS image
 POSTGIS_IMAGE = 'github.com/CanalTP/docker-postgis.git'
 POSTGIS_IMAGE_NAME = 'postgis:2.1'
 
 
 class DbParams(object):
+    """
+    Class to store and manipulate database parameters.
+    """
+
     def __init__(self, host, dbname, user, password):
+        """
+        Constructor of DbParams.
+
+        :param host: the host name
+        :param dbname: the database name
+        :param user: the user name to sue for the database
+        :param password: the password of the user
+        :return:
```

Comment:
If it doesn't return anything, why not remove this line? Same for every other "return" alone in the file?

Reply:
I thought it was convention to always write the return for mypy ... but if I'm wrong, tell me, I will remove it everywhere it's not needed.